xml/net/server/protocol.xml

```diff
@@ -701,12 +781,15 @@
                 <!-- ??? -->
             </case>
             <case default="true">
-                <comment>In this case, reply_code is a session ID for account creation</comment>
+                <comment>
+                    In this case (reply_code > 9), reply_code is a session ID for account creation
+                </comment>
                 <field name="sequence_start" type="char"/>
             </case>
         </switch>
-        <!-- "OK" or "NO" -->
-        <field name="reply_string" type="string"/>
+        <field name="reply_string" type="string">
+            <comment>"OK" or "NO"</comment>
```

Comment:
Actually for reply_code > 9 this string is "GO" :P

Reply:
![image](https://user-images.githubusercontent.com/22178573/208287934-a5a2c3b6-61ef-4ea4-86f2-68209c6363e5.png)
Can't reproduce with the official game server.

Reply:
Okay I lied it's for account reply 3 :P

<img width="905" alt="image" src="https://user-images.githubusercontent.com/3664832/208423699-26bde509-a66d-4111-8b9c-835f915905a3.png">

Reply:
Thanks, tidied this up in 9076c875269fc224d47e866f715dc6a805a83988.